src/commands/moderation/prefixCommands/functions/modifyCategory.ts

```diff
@@ -0,0 +1,99 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommandCategory, Logger, makeEmbed, refreshSinglePrefixCommandCategoryCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Modify Category - No Connection',
+    description: 'Could not connect to the database. Unable to modify the prefix command category.',
+    color: Colors.Red,
+});
+
+const failedEmbed = (categoryId: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Category - Failed',
+    description: `Failed to modify the prefix command category with id ${categoryId}.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (category: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Category - Does not exist',
+    description: `The prefix command category ${category} does not exists. Can not modify it.`,
+    color: Colors.Red,
+});
+
+const successEmbed = (category: string, categoryId: string) => makeEmbed({
+    title: `Prefix command category ${category} (${categoryId}) was modified successfully.`,
+    color: Colors.Green,
+});
+
+const modLogEmbed = (moderator: User, category: string, emoji: string, categoryId: string) => makeEmbed({
+    title: 'Prefix command category modified',
+    fields: [
+        {
+            name: 'Category',
+            value: category,
+        },
+        {
+            name: 'Moderator',
+            value: `${moderator}`,
+        },
+        {
+            name: 'Emoji',
+            value: emoji,
+        },
+    ],
+    footer: { text: `Category ID: ${categoryId}` },
+    color: Colors.Green,
+});
+
+const noModLogs = makeEmbed({
+    title: 'Prefix Commands - Modified Category - No Mod Log',
+    description: 'I can\'t find the mod logs channel. Please check the channel still exists.',
+    color: Colors.Red,
+});
+
+export async function handleModifyPrefixCommandCategory(interaction: ChatInputCommandInteraction<'cached'>) {
+    await interaction.deferReply({ ephemeral: true });
+
+    const conn = getConn();
+    if (!conn) {
+        await interaction.followUp({ embeds: [noConnEmbed], ephemeral: true });
+        return;
+    }
+
+    const category = interaction.options.getString('category')!;
+    const name = interaction.options.getString('name') || '';
+    const emoji = interaction.options.getString('emoji') || '';
+    const moderator = interaction.user;
+
+    //Check if the mod logs channel exists
+    const modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS) as TextChannel;
+    if (!modLogsChannel) {
```

Comment:
```suggestion
    let modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS);
    if (!modLogsChannel || !modLogsChannel.isTextBased()) {
        modLogsChannel = null;
```
Might be safer to actually check if the channel is a text channel? Especially if we want to offer the codebase to other projects. Then, if it's not a text channel set it to null to prevent accidental attempts to send into a non-text channel.